stf/model/prediction/lightgbm/model/quantile.py

```diff
@@ -0,0 +1,75 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+import dill
+from ktpbase.log import logging
+
+
+class XGBQuantileModel():
```

Comment:
This seems a bit weird, A XGBQuantile model in the lightgbm folder. Are you sure this doe not conflict with the XGBQuantile model in the xgboost folder?

EDIT: looks like the xgboost quantile files are duplicated, i think its best to remove this file.

Reply:
I'll change it, thanks!

Reply:
It looks like this file is still here, not used and duplicate of `stf.model.prediction.xgb.model.quantile`.